docs/changelogs/vNEXT.rst

```diff
@@ -55,6 +55,13 @@ Note that custom constructors are still supported for the Rust library since
 specifying a static ``new`` method for a type is nothing more than a
 convention.
 
+Improvements to the debugger
+----------------------------
+Three ways to step through queries, step, over and out.
+Inspect the stack.
```

Comment:
Which stack?

Reply:
Yeah, I don't really wanna call it the trace stack or the query stack. Since the differences are subtle and mostly implementation details. If you want something more specific we should call it the "call stack".